res/database/Default/user/therob.xml

```diff
@@ -2927,7 +2927,7 @@
 				<job index="3" function="2" required="0" gender="1" />
 			</jobs>
 			<data flags="2" optional_flags="16" />
-			<groups target_groups="3" optional_target_groups="4" />
+			<groups target_groups="7" />
```

Comment:
ich glaube "optional_flags" wird derzeit auch noch nicht verwertet.
-> da das Feld "flags_optional" heisst.

Ich denke, wir sollten "target_groups_optional" daraus machen - und im Spiel wuerde ich das dann aehnlich "flagsOptional" bei der Produktion behandeln.

hab da eh gleich noch n Fehler gefunden -- das eben nicht zufaellig aus den optionalen gewaehlt wird... sondern generell.

Reply:
Was ist denn der Unterschied zwischen flags und flags_optional bzw. target_groups und target_groups_optional. Wären das dann Zielgruppen, die im produzierten Programm verpflichtend drin sind oder manchmal schon oder auch nicht? (und flags analog?)

Reply:
genau. Du kannst bspweise "Zielgruppe Hausfrauen" definieren, und Optional noch "Rentner" - und dann waere manchmal "Hausfrauen" und ein anderes Mal "Hausfrauen, Rentner" die Zielgruppe(n)

Reply:
OK, dann sollte dieser Commit rückgängig gemacht und das Property umbenannt werden.

Reply:
Ja ich bin an dem "flag" bzw "targetgroup" schon dran ... momentan is das noch in verschiedenen Klassen. Denke nur das Template sollte die "optionalen" kennen und beim Drehbucherstellen entsprechend "finalisieren".

Urspruenglich war das so gedacht, dass der Regisseur oder so, aus dem "optionalen" was rausnehmen koennte (also FSK18 usw.). Aber ich glaube, wir koennen das auch weglassen und einfach etwas mehr "Zufall" in den generierten Drehbuechern erlauben.

Reply:
Mein Gefühl ist auch, dass man da der Konfiguritis etwas Einhalt gebieten könnte. Es würde ja tatsächlich reichen, wenn man Pflichtwerte definiert. Wenn dann noch eine Zielgruppe oder ein Flag rangewürfelt wird, schadet das nichts.
Also: optional aus der Datenbank rauslassen.

Reply:
Naja ich weiss halt nicht, wie man das mit dem "ranwuerfeln" limitieren will. Manches Drehbuch soll halt nicht "Zufaellig" FSK18 werden koennen - ich will aber nicht fuer jedes Genre definieren wollen, was da "so moeglich" ist. Das kann ja der Drehbuch"erschaffer" machen.

Entsprechend hab ich nix gegen "optional" - und der derzeitige "neue" Code wuerde das auch erstmal "auswerten".

Reply:
Verstehe ich auch, aber man könnte sich ja beim Dranwürfeln der Flags auch auf CULT; TRASH; BMOVIE beschränken; bei Erotik, Thriller, Horror und Action könnte XRATED dazukommen.
Wie sind denn die Chancen, dass die optional-flags überhaupt genutzt werden?

Reply:
Ich habe diese Änderung rückgängig gemacht und die flags entsprechend korrigiert.